lib/Db/LocalMessage.php

```diff
@@ -60,11 +60,28 @@
  * @method setSmimeCertificateId(?int $smimeCertificateId)
  * @method bool|null getSmimeEncrypt()
  * @method setSmimeEncrypt (bool $smimeEncryt)
+ * @method int getStatus();
```

Comment:
```suggestion
 * @method int|null getStatus();
```

the db column is nullable